linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -93,7 +121,7 @@ stage ("Deploy stage") {
                                     PowerShellWrapper("""
                                         & '.\\scripts\\lis_hyperv_platform\\main-lis-rpm.ps1'
                                             -JobId '${it}-d-${BUILD_NUMBER}'
-                                            -InstanceName '${it}-d-${BUILD_NUMBER}'
+                                            -InstanceName '${it}-deploy'
```

Comment:
make the pipeline to be executed one at a time
disableConcurrentBuilds
Disallow concurrent executions of the Pipeline. Can be useful for preventing simultaneous accesses to shared resources, etc. For example: options { disableConcurrentBuilds() }

check this out:
https://jenkins.io/doc/book/pipeline/syntax/#options